src/status_im2/common/standard_authentication/forgot_password_doc/style.cljs

```diff
@@ -0,0 +1,6 @@
+(ns status-im2.common.standard-authentication.forgot-password-doc.style)
+
+(def forget-password-doc-container {:margin-right 16})
```

Comment:
this doesn't make sense, the name is longer than the style map itself, `forget-password` is redundant in the name, and by guidelines it's allowed to still have short styles in the view

Reply:
yep, I just copied this code over. These long names are a symptom that the code should have been initially in its own namespace.  Anyway, you're right, I will adjust it 👌